src/lua/init.c

```diff
@@ -420,6 +424,61 @@ static const char *lua_modules_preload[] = {
  * {{{ box Lua library: common functions
  */
 
+/*
+ * Retrieve builtin module sources, if available.
+ */
+static const char *
+tarantool_debug_getsources(const char *modname)
+{
+	for (size_t i = 0; lua_modules[i] != NULL; i += 2) {
+		const char *shortname = lua_modules[i];
+		const char *lua_code = lua_modules[i + 1];
+		assert(lua_code != NULL);
+		char fullname[48];
```

Comment:
Hmm, well, to avoid constants entirely I've put it this way:
```diff
diff --git a/src/lua/init.c b/src/lua/init.c
index 60ec7ba80..6df4d7b6e 100644
--- a/src/lua/init.c
+++ b/src/lua/init.c
@@ -433,7 +433,8 @@ tarantool_debug_getsources(const char *modname)
 		const char *shortname = lua_modules[i];
 		const char *lua_code = lua_modules[i + 1];
 		assert(lua_code != NULL);
-		char fullname[48];
+		char fullname[sizeof("@builtin/.lua") +
+			      sizeof("luarocks.admin.cmd.refresh_cache")];
 		snprintf(fullname, sizeof(fullname), "@builtin/%s.lua",
 			 shortname);
 		if (!strcmp(shortname, modname) || !strcmp(fullname, modname))

```

Which is producing lesser aligned 47 bytes, but now it's clear from where we got those numbers.